envs/example/allinone-centos/vagrant.yml

```diff
@@ -1,7 +1,7 @@
 default:
   memory: 512
   cpus: 1
-  box: centos/7
+  box: rhel7
```

Comment:
we should create new envs for allinone-rhel7 and leave allinone-centos operational for centos